CMakeLists.txt

```diff
@@ -227,6 +227,7 @@ set(
   BOOST_CHRONO_DONT_PROVIDES_DEPRECATED_IO_SINCE_V2_0_0
   BOOST_STACKTRACE_USE_ADDR2LINE
   BOOST_ASIO_NO_DEPRECATED
+  OPENSSL_API_COMPAT=0x10100000L
```

Comment:
Consider adding a comment above this line explaining the reason for this.

Reply:
Good point

Reply:
Done